src/swarm/neo/node/ConnectionHandler.d

```diff
@@ -471,9 +523,26 @@ class ConnectionHandler : IConnectionHandler
                 // Supported version codes.
                 if (auto rq = command in this.shared_params.requests.request_info)
                 {
-                    this.sendSupportedStatus(connection,
-                        SupportedStatus.RequestSupported);
-                    handleRequest(command, *rq, connection, init_payload);
+                    // TODO: when the old deprecated handlers are removed, this
+                    // logic can be moved into handleRequest.
+                    this.handleRequest(*rq, connection,
+                        {
+                            this.acquirer_dg(this.shared_params.shared_resources,
+                                ( Object request_resources )
+                                {
+                                    auto rq_handler =
+                                        this.emplace!(IRequestHandler)
+                                        (connection.emplace_buf, rq.class_info);
```

Comment:
Ah, cool :+1:. I'll never be able to intuitively figure out the fact that the `connection` is actually `RequestOnConn` :-(

Reply:
Yeah, we should probably go through and make the doc / variable names consistent, at some point.